server/app/repository/ReportingRepository.java

```diff
@@ -67,6 +68,7 @@ public ImmutableList<ApplicationSubmissionsStat> loadThisMonthReportingData() {
         .sqlQuery(
             "SELECT\n"
```

Comment:
question (blocking): I'm a little confused, not by your update, but by how this was always done. Since we have created `monthly_submissions_reporting_view` why aren't querying from that pre-computed data?

Reply:
This is just for the current month. The view queries from the time beginning.

Reply:
Gotcha